spec/lib/pickup_bot_spec.rb

```diff
@@ -0,0 +1,56 @@
+require "spec_helper"
+require 'telegram/bot'
+require 'pickup_bot'
+
+feature "user needs help" do
+  let(:bot) { Telegram::Bot::Client.new('fake-token') }
+  let(:pickup_bot) { PickupBot.new(bot) }
+
+  scenario "user asks for help" do
+    stub_request(:any, /api.telegram.org/).to_return(status: 200, body:"[]", :headers => {})
+    @user = Telegram::Bot::Types::User.new(user_params)
+    @chat = Telegram::Bot::Types::Chat.new(chat_params)
+    message = Telegram::Bot::Types::Message.new(message_params('help'))
+
+    pickup_bot.run(message)
+
+    expect(a_request(:post, "https://api.telegram.org/botfake-token/sendMessage")).to have_been_made.times(1)
```

Comment:
We could also check the body of this request to see if we're sending the right message back.